### PR TITLE
Remove already deprecated stuff

### PR DESCRIPTION
**Pull request by fishy**
Also undeprecated redisbp into redis/db/redisbp.

Fixes https://github.com/reddit/baseplate.go/issues/363.

### CODE REVIEW

File: redis/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+# Redis packages in Baseplate.go
```
Comment (fishy) [inline redis/README.md:1]: @pacejackson please take a look at this file.

**Review by konradreiche — APPROVED** [review]

File: redis/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+## Use Redis as a cache
+
+If you use Redis as a cache, it's recommended to use
+[`redispipebp`][redispipebp] + [`redisx`][redisx] packages.
+They are built on top of [`redispipe`][redispipe] open source project.
+
+This combination can also be used if you use Redis as a DB but only do readings
+and not writings.
+
```
Comment (kylelemons) [inline redis/README.md:13]: you mention in the DB section why that's good for the DB use-case, but you don't mention why redispipe is good for the cache use-case.

Just a single sentence is probably fine, like "redispipe has improved support for pipelining reads to maximize throughput" or something (I'm making that up, I don't know if that's what it is)

**Review by kylelemons — APPROVED** [review]

File: redis/README.md (hunk 2 above)
Comment (fishy) [reply redis/README.md:13]: I'll wait for @pacejackson to provide a description :)

File: redis/README.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+[redisx]: https://pkg.go.dev/github.com/reddit/baseplate.go/redis/cache/redisx
+[redispipe]: https://github.com/joomcode/redispipe
+
+## Use Redis as a DB
+
+If you use Redis as a DB, and do writings,
+it's recommended to use [`redisbp`][redisbp] package.
+It's built on top of [`go-redis`][go-redis] open source project,
+and provides [`Wait`][wait] function to help you make sure to reach the desired
+consistency level on write operations.
```
Comment (pacejackson) [inline redis/README.md:24]: It might also be worth mentioning that you would want to use this if you are using redis as a queue or using pubsub.

Comment (fishy) [reply redis/README.md:24]: @pacejackson  Added a paragraph below. PTAL.

**Pull request merged**